metoo-sea.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: #MeToo In Seattle Resources
+permalink: /metoo-sea/
```

Comment:
Oh cool, I didn't realize jekyll supported multi-level paths here. I'm sticking with `metoo` since it corresponds to the #MeToo hashtag but I'm willing to change it if it's really a problem